torch/_functorch/_aot_autograd/collect_metadata_analysis.py

```diff
@@ -632,7 +632,11 @@ def view_avoid_dupes_with_primals(t):
                     t, lambda _, inner_t: view_avoid_dupes_with_primals(inner_t)
                 )
             if isinstance(t, Tensor):
-                return t.view(t.shape)
+                out = t.view(t.shape)
+                # Invariant: A fake tensor post-creation is guaranteed to
+                # have any associated symbolic nested int set
+                out.nested_int_memo = t.nested_int_memo  # type: ignore[attr-defined]
```

Comment:
it seems weird to have to set it manually here (or any places where we manually view out an intermediate tensor during tracing). Is there reason not to have NJT's view implementation in torch_dispatch automatically propagate the nested int?

Reply:
Hmm NJT's view should propagate it yes, though I think in this case we are calling "view" on the inner tensors, so the NJT logic is not triggered. 

Btw is the reason we don't just call "view" on the outer tensor because we cannot assume that subclass has implemented view?

Reply:
I actually cannot remember why don't just call `view()` directly in the tensor subclass case (doing it feels pretty reasonable). Do you want to try that and see if any tests fail?

Reply:
OH whoops I remember now, we purposely make sure the offsets is a different Tensor object lol
"[AOT Autograd: Views to avoid tangents aliasing inputs]"

Reply:
discussed offline. ideas:

(1) maybe change this call to an `aten.alias()` or `detach()`, since we know that the shape won't change?

(2) make the faketensor rule for alias/detach also propagate the nested int memo automatically, so users don't need to remember to do it